app/styles/partials/nav-bar.scss

```diff
@@ -0,0 +1,4 @@
+.right.menu > .item > .ui.blue.button {
```

Comment:
@CosmicCoder96 I misunderstood first-time @geekyd thanks for explaining.done changes please review.thanks